### ISSUE TITLE
Add binary agent for LicheeRV nano board

### ISSUE THREAD

**Issue by leleb** [Ylianst/MeshCentral#6590]
I just got one of these remote KVM:
https://github.com/sipeed/NanoKVM
and it would be very usefull if meshcentral agent could be installed in it.
Tried al binaries but none working.
It is based on LicheeRV nano board ( https://wiki.sipeed.com/hardware/en/lichee/RV_Nano/1_intro.html )

**Comment by si458** [comment]
sadly without one to test with, its going to be hard 👎 
if you happy to purchase another one OR donate some money towards one, then please get in contact!
https://www.si458.co.uk/2024/01/05/donation/ ❤️
i can then get one and look into it 👍

**Comment by leleb** [comment]
> sadly without one to test with, its going to be hard 👎 if you happy to purchase another one OR donate some money towards one, then please get in contact! https://www.si458.co.uk/2024/01/05/donation/ ❤️ i can then get one and look into it 👍

Hope you can soon buy one with my paypal donation. (Also there is cheaper lite version). Sadly at moment amazon or aliexpres require 2/3 weeks

**Comment by si458** [comment]
@leleb yes i did thank you! i did also emailing u asking if u was ok with the 4 week wait?
then realised u replied here saying the was a await anyways haha
so will get it ordered!
